projects/step-frontend/src/lib/modules/execution/components/alt-report-node-table-content/base-alt-report-node-table-content.component.ts

```diff
@@ -62,7 +64,12 @@ export abstract class BaseAltReportNodeTableContentComponent implements ItemsPer
   protected setupDateRangeFilter(): void {
     combineLatest([this._state.dateRange$, this.isRemoteDataSource$])
       .pipe(
-        map(([dateRange, isRemote]) => {
+        map(([pickerSelection, isRemote]) => {
+          const absoluteSelection = pickerSelection.absoluteSelection!;
+          const dateRange: DateRange = {
```

Comment:
@lucianbaciu There is `date-utils.service` that provides methods for TimeRange/DateRange and vice versa conversions. I this it make sense to reuse it here, instead of duplicating this logic